### PR TITLE
Coin Table

### PR DESCRIPTION
**Pull request by felipauskas**
- Added Coin Table
- Simple Styling

### CODE REVIEW

File: src/Components/CoinTable/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            this.setState({isLoading: false, hasError: true, hasData: false});
+        }
+    }
+
+    componentDidMount() {
+        this.getData();
+    }
+
+    componentDidUpdate() {
+        console.log(this.state);
```
Comment (yoloOnTheBattlefield) [inline src/Components/CoinTable/index.js:32]: meh

File: src/Components/CoinTable/index.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    render() {
+        const {coinData, hasData} = this.state;
```
Comment (yoloOnTheBattlefield) [inline src/Components/CoinTable/index.js:36]: const {` `coinData, hasData` `} = this.state;

File: src/Components/CoinTable/index.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            {hasData && (
+                <TableDiv>
+                    <CoinTableTitle />
+                    {coinData.map((element) => 
+                        <CoinTableData 
+                        key={element.market_cap_rank}
+                        {...element}
+                        hourchange={element.price_change_percentage_1h_in_currency}
+                        daychange={element.price_change_percentage_24h_in_currency}
+                        weekchange={element.price_change_percentage_7d_in_currency}
```
Comment (yoloOnTheBattlefield) [inline src/Components/CoinTable/index.js:48]: this is useless if you are spreading the element

File: src/Components/CoinTableData/chart.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { Line } from "react-chartjs-2";
+import {Chart, CategoryScale, Filler, LinearScale, PointElement, LineElement, BarElement, Title, Tooltip, Legend} from "chart.js";
```
Comment (yoloOnTheBattlefield) [inline src/Components/CoinTableData/chart.js:3]: import {` `Chart, CategoryScale, Filler, LinearScale, PointElement, LineElement, BarElement, Title, Tooltip, Legend` `} from "chart.js";

File: src/Components/CoinTableData/index.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+import { convertToMoney } from '../../utils';
+import { Name, CoinSlider, CoinDisplayDiv, Image, Rank, Hour, Price, Day, Week, VolumeMarket, CirculatingSupply, Last7d, ImgName, Volume, Market, Circulating, Supply, Completion } from './styles';
+import millify from 'millify';
+import SparklineChart from './chart';
+
+
+export default function CoinTableData(props) {
+    const {market_cap_rank, image, name, current_price, hourchange,
+            daychange, weekchange, total_volume, market_cap,
+            circulating_supply, total_supply, sparkline_in_7d} = props;
```
Comment (yoloOnTheBattlefield) [inline src/Components/CoinTableData/index.js:11]: this is ugly, each variable should on one line

File: src/Components/CryptoChart/index.js [hunk 6]
```diff
@@ -3,7 +3,8 @@ import axios from "axios";
 import { Line, Bar } from "react-chartjs-2";
 import {Chart, CategoryScale, Filler, LinearScale, PointElement, LineElement, BarElement, Title, Tooltip, Legend} from "chart.js";
 import { CurrentDate, ChartDiv, DataLabel, DataValue } from './styles';
-import { lineChartOptions, barChartOptions, convertToMoney } from './utils'
+import { convertToMoney } from '../../utils';
+import { lineChartOptions, barChartOptions } from './utils'
```
Comment (yoloOnTheBattlefield) [inline src/Components/CryptoChart/index.js:7]: rename this file to `chartUtils`

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/Components/CoinTable/index.js (hunk 3 above)
Comment (felipauskas) [reply src/Components/CoinTable/index.js:48]: I'm doing it to make the name shorter.

**Pull request merged**